.github/workflows/cd-update-table-metadata.yaml

```
@@ -0,0 +1,33 @@
+  push:
+    branches:
+      - main
+
+jobs:
+  build-container:
+    name: Update table metadata
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)